vm/ubpf_jit_x86_64.h

```diff
@@ -197,6 +198,18 @@ emit_alu32_imm8(struct jit_state *state, int op, int src, int dst, int8_t imm)
     emit1(state, imm);
 }
 
+/* Emit instructions to perform a blind load of the immediate value. */
+static inline void
+emit_alu32_blind_load_imm_to_rcx(struct jit_state *state, uint32_t imm)
+{
+    uint32_t secret = (uint32_t)random();
```

Comment:
Question: where is srandom() called to seed the RNG?

Reply:
Added code seed random in test.c startup.